### PR TITLE
Promote linux-aarch64 to a secondary platform

### PR DESCRIPTION
**Pull request by tom-cosgrove-arm**
It is regularly tested through the project CI on a system provided by Arm, and I support it (with my colleagues @zorrorffm and @xkqian)

### CODE REVIEW

File: policy-supplemental/platforms.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 | vms-ia64-p32 |        | OpenVMS 8.4 |        | ia64         |        | VSI C 7.4 [^1]  |        | \@levitte              |
 | vms-ia64-p64 |        | OpenVMS 8.4 |        | ia64         |        | VSI C 7.4 [^2]  |        | \@levitte              |
 | vms-x86\_64  |        | OpenVMS 8.4 |        | x86\_64      |        | VSI C X7.4 [^3] |        | \@levitte              |
+| linux-aarch64 |       | Linux       |        | AArch64      |        | gcc             |        | \@tom-cosgrove-arm     |
```
Comment (mattcaswell) [inline policy-supplemental/platforms.md:23]: I wonder if it is worth adding an additional column of "other maintainers". We are losing some useful information here.

Comment (tom-cosgrove-arm) [reply policy-supplemental/platforms.md:23]: (and remind me: additional commit, or force push with such an update?)

Comment (mattcaswell) [reply policy-supplemental/platforms.md:23]: > Would a column "Nominated Community Maintainers" make sense?

That works for me

> (and remind me: additional commit, or force push with such an update?)

I'd do it as a fixup commit. But either way.

**Review by mattcaswell — APPROVED** [review]

**Review by levitte — APPROVED** [review]

**Comment by mattcaswell** [comment]
Pushed.

**Pull request closed without merging**